### PR TITLE
Added Yash Pandey in members

### LINKED ISSUE #1558: Member profile in /community/members for Yash Pandey
**Current Behavior**
Yash Pandey currently doesn't have a community member profile/card

**Desired Situation**
Creation of community member profile/card
Yash Pandey

Yash Pandey

Here is the exported asset 
https://drive.google.com/file/d/1jo43k3xaB0jZCmaBTGcczRkaZOZ6D_CG/view
https://drive.google.com/file/d/167Vnea3TOHp7XCdnFikQnwB3X0zewU9I/view?usp=sharing
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
**Description**

This PR fixes #1558 .

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@96RadhikaJadhav thanks for this. This website was recently upgraded and uses a new member profile template. Will you confirm that this template is what you based this PR on? https://github.com/layer5io/layer5/blob/master/src/collections/members/_member-profile-template/index.mdx

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav thanks for this. This website was recently upgraded and uses a new member profile template. Will you confirm that this template is what you based this PR on? https://github.com/layer5io/layer5/blob/master/src/collections/members/_member-profile-template/index.mdx

@leecalcote Fixed changes according to new template.

File: collections/_members/yash-pandey.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: person
```
Comment (Tanuj22) [inline collections/_members/yash-pandey.md:2]: Remove this

File: collections/_members/yash-pandey.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+name: Yash Pandey
+position: Contributor
+image_path: /assets/images/community/members/yash-pandey.jpg
```
Comment (Tanuj22) [inline collections/_members/yash-pandey.md:5]: Can you keep this image in the same folder?

File: collections/_members/yash-pandey.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+github: EmperorYP7
+twitter: pandeyyash04
+linkedin: emperoryp
+bio: A passionate sophomore at Indian Institute of Information Technology, Ranchi pursuing Bachelors in Computer Science and Engineering trying to create an impact on open-source community by helping in making great softwares to empower people around the globe. I am interested to contribute and learn about the web and love to work on tools related to computer graphics.
+blurb: Mesh Head
+status: Active
+---
```
Comment (Tanuj22) [inline collections/_members/yash-pandey.md:12]: Add `published: true`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Thank you @96RadhikaJadhav . Can you do the following changes

File: collections/_members/yash-pandey.md (hunk 2 above)
Comment (96RadhikaJadhav) [reply collections/_members/yash-pandey.md:5]: @Tanuj22  Same folder as in??

Comment (Jashpatel1) [reply collections/_members/yash-pandey.md:5]: Keep the image in the same folder as index.mdx file

Comment (96RadhikaJadhav) [reply collections/_members/yash-pandey.md:5]: Done.

Comment (Tanuj22) [reply collections/_members/yash-pandey.md:5]: > @Tanuj22 Same folder as in??

Sorry just seeing this. @96RadhikaJadhav you need to update your master according to the latest fork. You have made a PR from the old site so this is creating a folder `collections/_members` whereas it should be in `src/collections/members/`

Comment (96RadhikaJadhav) [reply collections/_members/yash-pandey.md:5]: @Tanuj22 @Jashpatel1 I have updated master but somehow, the changes were not being reflected to my branch. So I created another branch and pushed the changes. Can you please close this PR and check #1607 .

**Comment by Jashpatel1** [comment]
Closing due to duplicancy.

**Pull request closed without merging**